src/mumble/CustomElements.cpp

```diff
@@ -69,6 +69,57 @@ void LogTextBrowser::scrollLogToBottom() {
 	verticalScrollBar()->setValue(verticalScrollBar()->maximum());
 }
 
+void LogTextBrowser::mouseMoveEvent(QMouseEvent *e) {
+	do {
+		QTextCursor cursor = cursorForPosition(e->pos());
+		QTextCharFormat fmt = cursor.charFormat();
+		if (fmt.objectType() == QTextFormat::NoObject) {
+			cursor.movePosition(QTextCursor::NextCharacter);
+			fmt = cursor.charFormat();
```

Comment:
Taken from https://github.com/mumble-voip/mumble/blob/master/src/mumble/MainWindow.cpp#L718 (which in turn was probably taken from http://stackoverflow.com/a/18901483).